.github/workflows/system_tests.yml

```diff
@@ -157,27 +151,31 @@ jobs:
         retry_failures: true
         
     - name: "Set PR Status Success"
-      if: ${{ success() && steps.findPr.outputs.number }}
-      uses: niteoweb/pull_request_status_action@v1.0.0
+      if: ${{ success() }}
+      uses: 0chain/actions/notify-pull-request@feature/notify-PR-convenience
+      continue-on-error: true
       with:
-        pr_number: ${{ steps.findPr.outputs.pr }}
-        description: "Manual system tests success"
         state: ${{ job.status }}
-        repository: ${{ github.repository }}
-        context: "0Chain System Tests"
-        target_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-      env:
-        GITHUB_TOKEN: ${{ github.token }}
+        message: "manual tests failed"
```

Comment:
could we send more detail message. It is better to send system_tests message with those branch/images whatever it is success or failed. Because we need know which branch/images are used . it helps us know whether PR should be merged.

Reply:
@stewartie4